services/events/events_definitions/models.go

```diff
@@ -0,0 +1,31 @@
+package events_definitions
+
+import "github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+
+type EventsDefinition struct {
+	ID             string       `json:"id"`
+	Name           string       `json:"name"`
+	Source         string       `json:"source"`
+	State          string       `json:"state"`
+	Description    string       `json:"description,omitempty"`
+	Mappings       []Mappings   `json:"mappings,omitempty"`
+	Rules          []Rules[any] `json:"rules,omitempty"`
+	CreatedAt      string       `json:"createdAt"`
+	CreatedBy      string       `json:"createdBy"`
+	LastModifiedAt string       `json:"lastModifiedAt"`
+	LastModifiedBy string       `json:"lastModifiedBy"`
```

Comment:
I don't see Tags

Reply:
I don't think so no, it's like CreatedBy it's common (to almost everything) but you have to add it each time